docs/pages/_app.js

```diff
@@ -275,6 +280,20 @@ function findActivePage(currentPages, pathname) {
   return activePage;
 }
 
+// Cache for the ltr version of the styles
+const cacheLtr = createCache({
+  key: 'mui',
+  stylisPlugins: [],
+  speedy: true,
+});
+
+// Cache for the rtl version of the styles
+const cacheRtl = createCache({
+  key: 'muirtl',
```

Comment:
Is using `mui` working? In https://emotion.sh/docs/@emotion/cache#key I'm not sure if it mean same DOM or VM.

Reply:
It is working bit note that these changes are only for the docs... I like when expecting the element I see mui-some_classname as I know immediately that it comes from emotion. Also we have two caches on the page, not at the same time but still I am worried if the styles will conflict. We can test it anyway